cURL/Endpoint Examples/JSON Payload/pdf-with-redacted-text-preview.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+UPLOAD_ID=$(curl --location 'https://api.pdfrest.com/upload' \
+--header 'Api-Key: xxxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx' \
+--header 'content-filename: filename.pdf' \
+--data-binary '@/path/to/file' \
+ | jq -r '.files.[0].id')
+
+echo "File successfully uploaded with an ID of: $UPLOAD_ID"
+
+REDACTIONS='[{\"type\":\"preset\",\"value\":\"uuid\"},{\"type\":\"regex\",\"value\":\"(\\\\+\\\\d{1,2}\\\\s)?\\\\(?\\\\d{3}\\\\)?[\\\\s.-]\\\\d{3}[\\\\s.-]\\\\d{4}\"},{\"type\":\"literal\",\"value\":\"word\"}]'
```

Comment:
```suggestion
REDACTIONS='[{\"type\":\"preset\",\"value\":\"email\"},{\"type\":\"regex\",\"value\":\"(\\\\+\\\\d{1,2}\\\\s)?\\\\(?\\\\d{3}\\\\)?[\\\\s.-]\\\\d{3}[\\\\s.-]\\\\d{4}\"},{\"type\":\"literal\",\"value\":\"word\"}]'
```

I get this error for this set of options. Replacing with `email` (or some other valid option) worked.

```
{"error": "uuid is not a valid preset redaction option. See documentation for valid options."}
```